app/views/home/_dmca.html.erb

```diff
@@ -80,7 +80,7 @@
     In the event that material is removed due to a <acronym title="Digital Millennium Copyright Act">DMCA</acronym> notice, the only recourse for restoring such material is to file a counter-notice with the <acronym title="Organization for Transformative Works">OTW</acronym>. If you believe that a take-down notice which has been acted upon by the <acronym title="Organization for Transformative Works">OTW</acronym> is without legal basis, please feel free to visit the following sites as a first step in learning about filing a counter-notice:
   </p>
   <ol>
-    <li> <a href="http://www.citmedialaw.org/legal-guide/responding-dmca-takedown-notice-targeting-your-content">"Responding to a <acronym title="Digital Millennium Copyright Act">DMCA</acronym> Takedown Notice Targeting Your Content"</a> at Citizen Media Law Project</li>
+    <li> <a href="http://www.dmlp.org/legal-guide/responding-dmca-takedown-notice-targeting-your-content">"Responding to a <acronym title="Digital Millennium Copyright Act">DMCA</acronym> Takedown Notice Targeting Your Content"</a> at Citizen Media Law Project</li>
```

Comment:
"Citizen Media Law Project" at the end of this line should now say "Digital Media Law Project"